Remotion/Security/Core/Metadata/ReflectionBasedMemberResolver.cs

```diff
@@ -144,9 +144,9 @@ private static bool TypeHasMember (Type type, string methodName, BindingFlags bi
       return existingMembers.Length > 0;
     }
 
-    private static bool IsSecuredMethod (MemberInfo memberInfo, object filterCriteria)
+    private static bool IsSecuredMethod (MemberInfo memberInfo, object? filterCriteria)
     {
-      string memberName = (string) filterCriteria;
+      var memberName = (string?) filterCriteria;
```

Comment:
I think filterCriteria would not be allowed to be null in our usecase?It's just the signature of FindMembers that makes it nullable. Should we add a debug-nullguard?